azure/item.go

```diff
@@ -47,6 +48,23 @@ func (i *item) Open() (io.ReadCloser, error) {
 	return i.client.GetBlob(i.container.id, i.id)
 }
 
+func (i *item) Partial(length, offset int64) (io.ReadCloser, error) {
```

Comment:
For my use-case, I need `io.ReadCloser`. I am trying to use Stow to allow backup to any cloud storage. https://github.com/restic/restic/pull/885